supervisor/addons/__init__.py

```diff
@@ -242,6 +244,8 @@ async def uninstall(self, slug: str) -> None:
         self.data.uninstall(addon)
         self.local.pop(slug)
 
+        self.sys_homeassistant.websocket.supervisor_update_event("store", {})
+        self.sys_homeassistant.websocket.supervisor_update_event("supervisor", {})
```

Comment:
Let's add this to job decorator

Reply:
We can't add tasks for specific methods to a generic decorator.